tox.ini

```diff
@@ -57,6 +57,7 @@ commands =
 [testenv:pypy]
 basepython = pypy
 commands =
+    pip install PyPyDispatcher>=2.0.6
```

Comment:
Looks like adding environment markers might result in issues with old setuptool (https://github.com/Lukasa/hyper/issues/279) but it looks like more and more packages start to require relatively recent setuptools and use environment markers (e.g. requests: https://github.com/requests/requests/issues/4006). But support of different kinds of markers varies, I'll check which versions are required in our case.